lightning/src/ln/mod.rs

```diff
@@ -86,6 +86,14 @@ pub struct PaymentPreimage(pub [u8;32]);
 /// (C-not exported) as we just use [u8; 32] directly
 #[derive(Hash, Copy, Clone, PartialEq, Eq, Debug)]
 pub struct PaymentSecret(pub [u8;32]);
+/// recipient-info type, use to provide payment_secret or bolt11 payment_metadata
```

Comment:
These need to be filled out a bit more - something about how this is arbitrary data to be communicated to the recipient - mostly came from the invoice. "recipient-info type" doesn't tell me much, maybe drop that?

Reply:
👍  
makes sense, was trying to match the style from the comment for PaymentSecret, but yeah, redundant. I'll take a stab at some of these comments with a fresh fixup commit and maybe we can iterate on them over github?